spring-mvc/src/main/resources/Texts_sk.properties

```diff
@@ -43,6 +43,17 @@ user.detail.badgeFromGym=Odznak {0} zo \u0161tadi\u00F3nu v meste {1}
 user.detail.usersFights=Zobrazi\u0165 tr\u00E9nerove s\u00FAboje
 user.detail.blockUser=Zablokova\u0165 tohto u\u017E\u00EDvate\u013Ea
 user.detail.unblockUser=Odblokova\u0165 tohto u\u017E\u00EDvate\u013Ea
+user.detail.choosePokemons=Vybra\u0165 in\u00FDch akt\u00EDvnych pok\u00E9monov
+user.detail.activePokemons=Tr\u00E9nerovi akt\u00EDvni pok\u00E9moni
+
+user.reorder.infoMessage=Zvo\u013E 6 akt\u00EDvnych pok\u00E9monov, ktor\u00ED bud\u00FA pou\u017Eit\u00ED na s\u00FAboje
+user.reorder.submitButton=Zvoli\u0165 6 akt\u00EDvnych pok\u00E9monov
+user.reorder.selectPokemon1=Vyber prv\u00E9ho pok\u00E9mona
```

Comment:
this could be done much nicer! :smile: and you know it! What will happen, when there are more than 6 active Pokemon? rewriting localizations is not very productive